libs/sql-schema-describer/src/walkers.rs

```diff
@@ -454,8 +454,9 @@ impl<'a> EnumWalker<'a> {
     }
 
     /// The values of the enum
-    pub fn values(self) -> &'a [String] {
-        &self.get().values
+    pub fn values(self) -> impl ExactSizeIterator<Item = &'a str> {
+        range_for_key(&self.schema.enum_variants, self.id, |(enum_id, _)| *enum_id)
+            .map(move |idx| self.schema.enum_variants[idx].1.as_str())
```

Comment:
This is a bit awkward to read. I'm not a huge fan of unnamed tuples. especially when it's not trivially obvious what they are.

Reply:
Yup, we could change from a tuple to `struct EnumVariant { enum_id: EnumId, variant_name: String }`.

Reply:
Done, and pushed.